### PR TITLE
Fix issues with artifact transfer in CI

### PR DESCRIPTION
**Pull request by rsynek**
Should fix the connection reset issues with transferring dependencies.

### CODE REVIEW

**Comment by triceo** [comment]
Doesn't appear to be fixed - in fact, quite the contrary.

**Comment by rsynek** [comment]
@triceo ironic, isn't it? It seems that `.` in a property name has a special meaning in PowerShell, which runs the command on Windows.

**Comment by rsynek** [comment]
The CI passed, let me re-run it a couple of times to increase the probability that the issue is gone.

File: .github/workflows/jpa_integration_ci.yml [hunk 1]
```diff
@@ -10,6 +10,13 @@ on:
     branches:
       - master
 
+env:
+  MVN_CONNECTION_PROPS: -Dmaven.wagon.httpconnectionManager.ttlSeconds=60 -Dmaven.wagon.http.retryHandler.count=5
```
Comment (triceo) [inline .github/workflows/jpa_integration_ci.yml:14]: I'd add a comment here, explaining why those are necessary.

Comment (rsynek) [reply .github/workflows/jpa_integration_ci.yml:14]: good idea

**Pull request merged**